src/ontology/edits/oeo-co-sim.omn

```diff
@@ -0,0 +1,162 @@
+Prefix: : <http://openenergyplatform.org/ontology/oeo/oeo-co-sim/>
+Prefix: dc: <http://purl.org/dc/elements/1.1/>
+Prefix: owl: <http://www.w3.org/2002/07/owl#>
+Prefix: rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+Prefix: rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+Prefix: xml: <http://www.w3.org/XML/1998/namespace>
+Prefix: xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+
+Ontology: <http://openenergyplatform.org/ontology/oeo/oeo-co-sim/>
+<http://purl.obolibrary.org/obo/http://openenergyplatform.org/ontology/oeo/oeo-co-sim//http://openenergyplatform.org/ontology/oeo/dev/oeo-co-sim.omn/http://openenergyplatform.org/ontology/oeo/oeo-co-sim/.owl>
+Import: <http://openenergyplatform.org/ontology/oeo/dev/oeo-model.omn>
+
+Annotations: 
+    dc:description "The Open Energy Ontology is an ontology for all aspects of the energy modelling domain. The 'Co-Sim' module covers entities for models, assumptions, parameters, data, software and all the processes that transform these, including model execution and data transformation.",
+    <http://purl.org/dc/terms/license> "http://creativecommons.org/publicdomain/zero/1.0/",
+    <http://purl.org/dc/terms/title> "Open Energy Ontology (Modelling module)",
+    <http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion> "1.2"
+
+AnnotationProperty: <http://purl.obolibrary.org/obo/IAO_0000115>
+
+    
+AnnotationProperty: <http://purl.obolibrary.org/obo/IAO_0000118>
+
+    
+AnnotationProperty: <http://purl.org/dc/terms/license>
+
+    
+AnnotationProperty: <http://purl.org/dc/terms/title>
+
+    
+AnnotationProperty: <http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion>
+
+    Annotations: 
+        rdfs:label "has_obo_format_version"
+    
+    
+AnnotationProperty: dc:description
+
+    
+AnnotationProperty: rdfs:label
+
+    
+Datatype: rdf:PlainLiteral
+
+    
+Datatype: xsd:string
+
+    
+Class: <http://openenergyplatform.org/ontology/oeo/OEO_00420001>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An information content entity that is intended for use within simulation."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "Simulationskomponente"@de,
+        rdfs:label "simulation component"@de
```

Comment:
```suggestion
        rdfs:label "simulation component"@en
```
Please set the language to english in protege preferences.

Reply:
And update the language tag for the English labels to @en throughout your implementation